Data/Sys/GameSettings/STR.ini

```diff
@@ -0,0 +1,19 @@
+# STRE4Q - Tron: Evolution - Battle Grids (NTSC-U)
+# STRP4Q - Tron: Evolution - Battle Grids (PAL)
+# STRX4Q - Tron: Evolution - Battle Grids Championship Edition (NTSC-U)
```

Comment:
Please change this to

# STRE4Q, STRP4Q, STRX4Q - Tron Evolution - Battle Grids

This is more in-line with how we format GameINIs, I'm not on my computer, but you can check other 3 letter INIs to see how it is normally handled.  I know that one of these are a slightly different name, I think there's a Shawn White game or something with a similar edition, not sure how we handle that.

Reply:
Fixed, "Shaun White Snowboarding: Road Trip" has a different code (the last letter is different) between the normal and Target Limited Edition so it has a different entry for both entries. For this game I've combined everything as you suggested.

Reply:
Sounds fine.  If we add 6 letter INI, we can put the full name for Championship Edition when it comes.